endpoints.json

```diff
@@ -75,5 +75,17 @@
         }
       ]
     }
+  },
+  "POST /api/articles/:article_id/comments": {
+    "description": "adds a new comment and serves it back",
+    "queries": [],
```

Comment:
When we do post or patch, we also need an exampleRequest. In this case it's gonna be {username : string, body: string}